src/ipc/capnp/mining.capnp

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2024 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+@0xc77d03df6a41b505;
+
+using Cxx = import "/capnp/c++.capnp";
+$Cxx.namespace("ipc::capnp::messages");
+
+using Proxy = import "/mp/proxy.capnp";
+$Proxy.include("interfaces/mining.h");
+$Proxy.includeTypes("ipc/capnp/mining-types.h");
+
+interface Mining $Proxy.wrap("interfaces::Mining") {
+    isTestChain @0 (context :Proxy.Context) -> (result: Bool);
+    isInitialBlockDownload @1 (context :Proxy.Context) -> (result: Bool);
+    getTipHash @2 (context :Proxy.Context) -> (result: Data);
+    createNewBlock @3 (context :Proxy.Context, scriptPubKey: Data, options: BlockCreateOptions) -> (result: CBlockTemplate);
```

Comment:
02c6a1ed58867fa14b1e0c42f476ef45e64ea7ea `-> (result: BlockTemplate);`

Reply:
re: https://github.com/bitcoin/bitcoin/pull/30510#discussion_r1762668345

> [02c6a1e](https://github.com/bitcoin/bitcoin/commit/02c6a1ed58867fa14b1e0c42f476ef45e64ea7ea) `-> (result: BlockTemplate);`

Thanks, should be updated in latest push.